cluster.c

```diff
@@ -770,10 +757,21 @@ RRStatus ShardingInfoValidateShardGroup(RedisRaftCtx *rr, ShardGroup *new_sg)
         }
 
         for (int i = new_sg->slot_ranges[h].start_slot; i <= new_sg->slot_ranges[h].end_slot; i++) {
-            if (si->hash_slots_map[i] != 0) {
-                LOG_ERROR("Invalid shardgroup: hash slot already mapped: %u", i);
+            if (si->stable_slots_map[i] != 0) {
+                LOG_ERROR("Invalid shardgroup: hash slot already mapped as stable: %u", i);
```

Comment:
Minor but just for type safety and silence linter, should `i` be unsigned here?

Reply:
done